code/modules/mob/living/simple_animal/abnormality/abno_cores.dm

```diff
@@ -0,0 +1,73 @@
+/obj/structure/abno_core
+	name = "blank abnormality core"
+	desc = "You shouldn't be seeing this. Please let someone know!"
+	icon = 'ModularTegustation/Teguicons/abno_cores/zayin.dmi'
+	icon_state = ""
+	anchored = FALSE
+	density = FALSE
+	resistance_flags = INDESTRUCTIBLE
+	pixel_x = -16
+	base_pixel_x = -16
+	var/mob/living/simple_animal/hostile/abnormality/contained_abno
+	var/release_time = 420 SECONDS//this is going to be reduced by a minute for every risk level
+	var/threat_level
+
+/obj/structure/abno_core/proc/Release()
+	if(!contained_abno)//Is this core properly generated?
+		return
+	new contained_abno(get_turf(src))
+	qdel(src)
+	return TRUE
+
+/obj/structure/abno_core/MouseDrop(over_object)
+	var/obj/machinery/abno_core_extractor/O = over_object//abno core extractor
+	if(istype(O) && Adjacent(O) && Adjacent(usr))
+		var/response = alert(usr,"Will you extract [src]?","This cannot be reversed.","Yes","No")
+		if(response == "Yes" && do_after(usr, 10, O))
+			Extract()
+			return
+	var/obj/structure/itemselling/I = over_object//item selling machine
+	if(istype(I) && Adjacent(I) && Adjacent(usr))
+		var/response = alert(usr,"Will you sell [src]?","This cannot be reversed.","Yes","No")
+		if(response == "Yes" && do_after(usr, 10, I))
+			if(!contained_abno || !threat_level)//is there no risk level or abnormality inside?
+				qdel(src)
+				return
+			var/payment_amt = ((threat_level *= threat_level) * 100)
```

Comment:
```suggestion
			var/payment_amt = threat_level * threat_level * 100
```
This is just threat_level squared times 100 right? *= i believe is incorrect and otherwise order of operations doesn't matter because it's all mult